code/datums/components/paintable.dm

```diff
@@ -19,8 +19,6 @@
     if(spraycan.is_capped)
         to_chat(user, "<span class='warning'>Take the cap off first!</span>")
         return
-    if(spraycan.check_empty(user))
-        return
     RemoveCurrentCoat()
     spraycan.use_charges(2)
```

Comment:
Where's the check to see if it has those 2 charges?

Reply:
Now use_charges does the check

Reply:
Wait shit i forgot to pass the user in use_charges dnm